delta.test.js

```diff
@@ -1,63 +1,51 @@
 const { test, expect } = require("@jest/globals");
-const { daysBetween } = require("./delta");
+const { daysBetween, isLeapYear } = require("./delta");
 
 test("calculates days between dates in same month", () => {
   const from = { year: 2020, month: 10, day: 2 };
   const to = { year: 2020, month: 10, day: 10 };
-
   expect(daysBetween(from, to)).toBe(8);
 });
 
-test("calculates days between year-long period containing leap year", () => {
-  const from = { year: 2018, month: 1, day: 2 };
-  const to = { year: 2021, month: 2, day: 3 };
-
-  expect(daysBetween(from, to)).toBe(1128);
-});
```

Comment:
No need to move this test to the bottom.

Reply:
Done